src/app/loaders/express/index.ts

```diff
@@ -166,6 +141,16 @@ const loadExpressApp = async (connection: Connection) => {
   // New routes in preparation for API refactor.
   app.use('/api', ApiRouter)
 
+  // Serve static client files only in prod
+  if (config.nodeEnv === Environment.Prod) {
+    const frontendPath = path.resolve('dist/frontend')
```

Comment:
where in the React config do we specify that the React build files should output to `dist/frontend`?

Reply:
in the build path flag in package.json `BUILD_PATH='../dist/frontend'`